### DISCUSSION TITLE
Something wrong with the offical website?

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#18787] [Q&A General] [answered]
Dear,
It seems that the offcial website of MOOSE (https://mooseframework.inl.gov/) has crashed? I can not access the website this morning til now.
Does that happen to you?

**Comment by WilkAndy** [comment]
Yep, i agree

**Comment by GiudGiud** [comment, marked as answer]
Hello

There are a few technical difficulties on some websites at the lab. We'll keep you posted once we know more. 
You can build the website locally in the mean time. In the doc/ of your app: `python moosedocs.py build --serve`

Guillaume

**Reply by ZhigangPu** [reply]
Wow, so cool, thanks very much!

Zhigang

**Reply by milljm** [reply]
Should be up and working now.

**Reply by ZhigangPu** [reply]
Yes, indeed, thanks for your effort!
